equinix/fabric_connection_read_schema.go

```diff
@@ -0,0 +1,708 @@
+		},
+		"deleted_by_email": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by User Email Address",
+		},
+		"deleted_date_time": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Deleted by Date and Time",
+		},
+	}
+}
+
+func readOrderSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"purchase_order_number": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Purchase order number",
+		},
+		"billing_tier": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Billing tier for connection bandwidth",
+		},
+		"order_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Order Identification",
+		},
+		"order_number": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Order Reference Number",
+		},
+	}
+}
+
+func readAccountSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"account_number": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Account Number",
+		},
+		"account_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Account Name",
+		},
+		"org_id": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Customer organization identifier",
+		},
+		"organization_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Customer organization name",
+		},
+		"global_org_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Global organization identifier",
+		},
+		"global_organization_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Global organization name",
+		},
+		"global_cust_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Global Customer organization identifier",
+		},
+	}
+}
+
+func readNotificationSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"type": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Notification Type",
+		},
+		"send_interval": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Send interval",
+		},
+		"emails": {
+			Type:        schema.TypeList,
+			Computed:    true,
+			Description: "Array of contact emails",
+			Elem: &schema.Schema{
+				Type: schema.TypeString,
+			},
+		},
+	}
+}
+
+func readErrorAdditionalInfoSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"property": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Property at which the error potentially occurred",
+		},
+		"reason": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Reason for the error",
+		},
+	}
+}
+
+func readOperationalErrorSch() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"error_code": {
```

Comment:
Same concern as above in `readErrorAdditionalInfoSch()`.

Reply:
Added explanation to the above comment.